python/ray/tests/test_client_builder.py

```diff
@@ -90,20 +90,20 @@ def ping(self):
 
 def test_connect_to_cluster(ray_start_regular_shared):
     server = ray_client_server.serve("localhost:50055")
-    client_info = ray.client("localhost:50055").connect()
-
-    assert client_info.dashboard_url == ray.worker.get_dashboard_url()
-    python_version = ".".join([str(x) for x in list(sys.version_info)[:3]])
-    assert client_info.python_version == python_version
-    assert client_info.ray_version == ray.__version__
-    assert client_info.ray_commit == ray.__commit__
-    protocol_version = ray.util.client.CURRENT_PROTOCOL_VERSION
-    assert client_info.protocol_version == protocol_version
+    with ray.client("localhost:50055").connect() as client_ctx:
+        assert client_ctx.dashboard_url == ray.worker.get_dashboard_url()
+        python_version = ".".join([str(x) for x in list(sys.version_info)[:3]])
+        assert client_ctx.python_version == python_version
+        assert client_ctx.ray_version == ray.__version__
+        assert client_ctx.ray_commit == ray.__commit__
+        protocol_version = ray.util.client.CURRENT_PROTOCOL_VERSION
+        assert client_ctx.protocol_version == protocol_version
 
     server.stop(0)
     subprocess.check_output("ray stop --force", shell=True)
 
 
+@pytest.mark.skip()
```

Comment:
Hmm, should this really be skipped?

Reply:
OOps, that was a debugging thing (I was trying to figure out why a test-test interaction was causing problems).